mm/2s2h/Rando/Menu.cpp

```diff
@@ -323,8 +323,8 @@ static void DrawItemsTab() {
                                        "that must be found in order for their corresponding boss to spawn.",
                             .disabled = IncompatibleWithLogicSetting(RO_SHUFFLE_BOSS_SOULS),
                             .disabledTooltip = "Incompatible with current Logic Setting" } }));
-    CVarCheckbox("Enemy Souls", "gPlaceholderBool",
-                 CheckboxOptions({ { .disabled = true, .disabledTooltip = "Coming Soon" } }));
+    CVarCheckbox("Enemy Souls", Rando::StaticData::Options[RO_SHUFFLE_ENEMY_SOULS].cvar,
+                 CheckboxOptions({ { .disabled = false, .disabledTooltip = "Coming Soon" } }));
```

Comment:
This should be marked as incompatible with vanilla and French vanilla. Clang format might hate my suggestion:

```suggestion
                 CheckboxOptions({ { .tooltip = "Adds the \"souls\" of regular enemies to the item pool. Enemy Souls are items "
                            "that must be found in order for their corresponding enemy to spawn.",
                            .disabled = IncompatibleWithLogicSetting(RO_SHUFFLE_ENEMY_SOULS),
                            .disabledTooltip = "Incompatible with current Logic Setting" } }));
```